packages/site/src/components/Header.tsx

```diff
@@ -69,13 +68,38 @@ export const Header = ({
       dispatch({ type: MetamaskActions.SetError, payload: e });
     }
   };
+
+  function Version() {
+    return (
+      <VersionStyle>
+        <div>
+          <b>dApp version: </b>
```

Comment:
I think that for consystency we should use:

```suggestion
          <b>Dapp version: </b>
```

Example: https://support.metamask.io/hc/en-us/articles/4405506066331-User-Guide-Dapps